src/modules/dpos/validator/hooks/queries/useForgersGenerator.test.js

```diff
@@ -6,8 +6,8 @@ import { useForgersGenerator } from './useForgersGenerator';
 
 jest.useRealTimers();
 
-describe('useBlocks hook', () => {
-  const limit = 15;
+describe('useForgersGenerator hook', () => {
+  const limit = 52;
```

Comment:
do we need all 52 of them, let's make test fast

Reply:
this was done since the length of the in round forger list is definite. Also using a limit of 15 resulted to unwanted complications in the last test assertion.